docs/hugo/content/guide/installing-from-helm.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Installation: From Helm"
+weight: -2
+menu:
+  main:
+    weight: -2
```

Comment:
I think we're getting too many things in the header menu here, we need to keep it down to around five:

<img width="1914" alt="image" src="https://user-images.githubusercontent.com/1272094/231856281-ececeafb-5f01-424f-b40a-e9963bf53de6.png">

Reply:
Fixed, just wanted ordered in the sidebar